### PR TITLE
Feat: user message for invalid RGB input

### LINKED ISSUE #76: Create user message for invalid inputs for rgb functionality
If the user does not input the proper format for rgb value then we should display an invalid format message.
We should also show them the correct format so they can input the correct value.

### PR DESCRIPTION
**Pull request by sparkO7**
# Description

Added two error messages, one displays if one of the inputs is empty and the other displays if the input value is invalid.
I also modified the result to only show when clicking away/out of the input (onblur), while the color displays on both swatches while typing (oninput).

Because of some code confusion, I also added a small regex check for the HEX part of the code.
(When typing the rgb value, code from the HEX part executes. Ex: `firstColor.length < 7` or `firstColor.length >= 7` ) 
\- A small fix for now.

Fixes #76 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore items (this includes basic clean up of files, or updates to packages)
- [ ] Updates to documentation

## Checklist

- [x] I have a descriptive title for my PR
- [x] I have linked this PR to the corresponding issue. [See how to do that here.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] I have run the project locally and reviewed the code changes
- [x] My changes generate no new warnings

### CODE REVIEW

File: index.js
Comment (AndreyDmitriev39r) [inline index.js:202]: OK. So you nested all hex stuff in the if check, that seemes like good option for now.
Maybe you can create variable for this regexp - but it is totally optional.

**Review by AndreyDmitriev39r — COMMENTED** [review]
Code looks good to me here.

But before testing it locally:
Looks like we have another merge conflict here(again).

So fix this one and I will proceed with the review.

**Review by AndreyDmitriev39r — APPROVED** [review]
Approving it.

Waiting for more reviews before merging.

Some temporary issues appeared because of these changes, but we are planning to deal with it  when working on error messages for other formats.

For details see comments in this thread from @sparkO7 and ask questions in discord

**Comment by sparkO7** [comment]
Some other changes I've made:
\- I replaced the rgb regex variable with a function and I suggest we make the same change for the other color formats.
\- I added a new function to display the color. So, the `displayResult` function will display the result only.

Some issues appeared while working on this issue, that I think we can fix later of course:
\- The empty error field shows before filling the second input.
\- The error displaying is only accurate for the RGB format, not for the other formats. So the invalid error message may appear when typing other color formats.
\- The added HEX regex check is only a fix for now.

**Review by jdwilkin4 — APPROVED** [review]
I think this is good to go in.
Since this is not in production yet, we can just create another ticket for the hsl and hex invalid inputs.

**Pull request merged**
